PyFlow/Core/AGraphCommon.py

```diff
@@ -16,19 +16,18 @@
 from enum import IntEnum
 import Enums
 
-
+## determines int maximum value
+INT_RANGE_MAX = 99999
```

Comment:
Why are you not using `sys.maxint`? If you use python3 there is no maxint anymore but here is a version that works for both (py 2 &3):

~~~python
import struct
maxint = 2 ** (struct.Struct('i').size * 8 - 1) - 1
~~~

Reply:
nice. will add this later

Reply:
Does this work cross platform? I know maxint is no longer a thing in py3 but the maxint was breaking on OSX as well as linux even in py2. So I just hardcoded that number in to see if that was the issue and then see if you guys had a suggestion for how to handle that issue. Thanks!